pkg/pb/nodestats.proto

```diff
@@ -22,3 +24,18 @@ message UptimeCheckResponse {
     double reputation_score = 5;
 }
 
+message DailyStorageUsageRequest {
+    google.protobuf.Timestamp from = 1;
+    google.protobuf.Timestamp to = 2;
+}
+
+message DailyStorageUsageResponse {
+    message StorageUsage {
+        double at_rest_total = 2;
+        google.protobuf.Timestamp time_stamp = 3;
```

Comment:
I remember @wthorp mentioned that we can use gogoproto.customtype serialization 'automatically' to time.Time instead 'manually' converting through google.protobuf.Timestamp. Seems like this could be a good place to use that?

Reply:
Didn't know about it